bin/airgap-setup-parsing.sh

```diff
@@ -0,0 +1,143 @@
+#! /bin/bash
+
+source bin/common.sh
+
+required_vars=(
+    "AIRGAP_REGISTRY"
+    "AIRGAP_REGISTRY_USERNAME"
+    "AIRGAP_REGISTRY_PASSWORD"
+)
+
+for var in "${required_vars[@]}"; do
+    if [ -z "${!var}" ]; then
+        log_error "Environment variable $var is not set."
+        log_error "Please set it using: export $var=VALUE OR..."
+        log_error "...Set it in USER_DIR/user.env file."
+        exit 1
+    fi
+done
+
+if [[ -n $AIRGAP_HELM_REPO && $AIRGAP_HELM_REPO != "$AIRGAP_REGISTRY" ]]; then
+    log_warn "AIRGAP_HELM_REPO ($AIRGAP_HELM_REPO) does not match AIRGAP_REGISTRY ($AIRGAP_REGISTRY)."
+    log_warn "This script currently does not support the storing of two types of different artifacts in different locations."
+    echo
+fi
+
+log_info "docker login ""$AIRGAP_REGISTRY"" -u ""$AIRGAP_REGISTRY_USERNAME"" -p ___"
+docker login "$AIRGAP_REGISTRY" -u "$AIRGAP_REGISTRY_USERNAME" -p "$AIRGAP_REGISTRY_PASSWORD"
+
+log_info "Parsing $COMPONENT_FILE"
+echo
+
+log_notice "Step 1 - Import Images"
+echo
+
+for var in $(compgen -A variable | grep '_FULL_IMAGE$'); do
+    full_image="${!var}"
+
+    [[ -z $full_image ]] && continue
+
+    if [[ $full_image == "registry.redhat.io/openshift4/ose-oauth-proxy:latest" ]]; then
+        log_warn "Skipping image: $full_image"
+        echo
+        continue
+    fi
+
+    parseFullImage "$full_image"
+
+    # shellcheck disable=2153
+    repo_image="$REPOS/$IMAGE:$VERSION"
+
+    log_verbose "docker pull ""${full_image}"""
+    docker pull "${full_image}"
+    echo
+    log_verbose "docker tag ""${full_image}"" ""$AIRGAP_REGISTRY""/""${repo_image}"""
```

Comment:
The `log_verbose` messages are suppressed if the `LOG_VERBOSE_ENABLE` environment variable is set to 'false'.  (The default value is 'true'.)  The idea is that some messages are helpful but not critical for the user to see and to provide a mechanism to cut down on the message generated.  Have you chosen to make these "verbose" messages with that in mind?  That may be a good idea, I just want to confirm that was the intent.